apps/workflows-dashboard/src/common/hooks/useSorting/useSorting.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  getSortingDataFromQuery,
+  SortingData,
+} from '@app/common/hooks/useSorting/helpers/get-sorting-data-from-query';
+import { buildSortingRegex } from '@app/common/hooks/useSorting/utils/build-sorting-regex';
+import { SortingParams } from '@app/common/types/sorting-params.types';
+import { useCallback, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+export function useSorting(prefix = 'order_by') {
```

Comment:
In the future please see if there's existing code and I'll make sure to send references. We already have logic for sorting and so on in the backoffice and it uses `qs`. 🙏